x-pack/plugins/security_solution/public/common/components/navigation/breadcrumbs/index.test.ts

```diff
@@ -128,7 +127,7 @@ describe('Navigation Breadcrumbs', () => {
       );
       expect(breadcrumbs).toEqual([
         {
-          href: '/app/security/overview',
+          href: 'securitySolutionoverview',
```

Comment:
nit: took me a second to figure out why this changed. I believe it's because of the mock on 112 that's doing:

```
const getUrlForAppMock = (appId: string, options?: { path?: string; absolute?: boolean }) =>
  `${appId}${options?.path ?? ''}`;	  `${appId}${options?.path ?? ''}`;
```

That used instead of this right?

```
getUrlForApp(APP_ID, { path: SecurityPageName.overview }),
```
from here: https://github.com/elastic/kibana/pull/87245/files#diff-8c78fbdebc67d5ff9388db4a66a58416997b3745367d7c49d88a7834d963eec2R77

Maybe a comment explaining why it's no longer `/app/security/overview`?

Reply:
Added a comment explaining that the mock function returns not an actual url.

Reply:
Yea, took me a sec to realize it was just replacing the `APP_OVERVIEW_PATH` const. Thanks